quickstarts/aws/aws-storage-gateway/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: aws-storage-gateway
+description: |-
+  ## What is AWS Storage Gateway?
+  AWS Storage Gateway is a set of hybrid cloud storage services that provide on-premises access to virtually unlimited cloud storage.
+  ### Get started!
+  Start monitoring AWS Storage Gateway by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-storage-gateway-integration/) to learn more about New Relic monitoring for AWS Storage Gateway.
```

Comment:
@aj-NR This docs url is not working

Reply:
@mdumpati Yes its not yet live. I have asked the team to merge it today. Thats the reason in PR I have not checked the docs live checkbox.